### PR TITLE
[zh-cn] sync scheduling-eviction/*

### PR DESCRIPTION
**Pull request by my-git9**
```
M       content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md
M       content/zh-cn/docs/concepts/scheduling-eviction/kube-scheduler.md
M       content/zh-cn/docs/concepts/scheduling-eviction/scheduler-perf-tuning.md
M       content/zh-cn/docs/concepts/scheduling-eviction/taint-and-toleration.md
M       content/zh-cn/docs/concepts/scheduling-eviction/topology-spread-constraints.md

```

### CODE REVIEW

File: content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md [hunk 1]
```diff
@@ -252,6 +253,20 @@ set aside for it.
...
 
+{{< note >}}
+<!--
+Scheduling of pods which use ResourceClaims is going to be slower because of
+the additional communication that is required. Beware that this may also impact
+pods that don't use ResourceClaims because only one pod at a time gets
+scheduled, blocking API calls are made while handling a pod with
+ResourceClaims, and thus scheduling the next pod gets delayed.
+-->
+由于需要额外的通信，使用 ResourceClaims 的 Pod 的调度将会变慢。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md:264]:
```suggestion
由于需要额外的通信，使用 ResourceClaim 的 Pod 的调度将会变慢。
```

File: content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md [hunk 2]
```diff
@@ -252,6 +253,20 @@ set aside for it.
...
+请注意，这也可能会影响不使用 ResourceClaims 的 Pod，因为一次仅调度一个
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md:265]:
```suggestion
请注意，这也可能会影响不使用 ResourceClaim 的 Pod，因为一次仅调度一个
```

File: content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md [hunk 3]
```diff
@@ -252,6 +253,20 @@ set aside for it.
...
+Pod，在使用 ResourceClaims 处理 Pod 时会进行阻塞 API 调用，
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md:266]:
```suggestion
Pod，在处理使用 ResourceClaim 的 Pod 时会进行阻塞 API 调用，
```

File: content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md [hunk 4]
```diff
@@ -252,6 +253,20 @@ set aside for it.
...
+从而延迟调度下一个 Pod。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md:267]:
```suggestion
从而推迟调度下一个 Pod。
```

**Review by windsonsea — COMMENTED** [review]
/lgtm

**Pull request merged**